src/openqaoa-qiskit/setup.py

```diff
@@ -0,0 +1,43 @@
+from setuptools import setup, find_packages
+from os import getcwd
+
+current_path = getcwd()
+
+# with open("README.md", "r") as fh:
+#     long_description = fh.read()
+
+long_description = ""
+
+with open("_version.py") as f:
+    version = f.readlines()[-1].split()[-1].strip("\"'")
+
+requirements = [
+    "openqaoa-core=={}".format(version),
+    "qiskit>=0.36.1",
```

Comment:
I don't think we need all of qiskit, right?

Reply:
Handle in a separate PR